server/data/visitSchedulerApiTypes.ts

```diff
@@ -6,7 +6,9 @@ export type CategoryGroup = components['schemas']['SessionCategoryGroupDto']
 // Incentive groups
 export type IncentiveLevelGroup = components['schemas']['SessionIncentiveLevelGroupDto']
 export type CreateIncentiveGroupDto = components['schemas']['CreateIncentiveGroupDto']
+export type CreateCategoryGroupDto = components['schemas']['CreateCategoryGroupDto']
 export type IncentiveLevels = IncentiveLevelGroup['incentiveLevels'][number]
+export type PrisonerCategories = CategoryGroup['categories'][number]
```

Comment:
These new types are fine but to follow the existing structure of the file they should be in the section above

Reply:
Updated